addon/services/intl.js

```diff
@@ -52,6 +52,10 @@ const IntlService = Service.extend(Evented, {
   formatTime: formatterProxy('time'),
   formatDate: formatterProxy('date'),
 
+  t(...args) {
+    return this.formatMessage(this.findTranslationByKey(...args));
```

Comment:
What is the current signature of the `{{t}}` helper? I'd like to maintain the same arguments that it supports.